gearcatalog/frontend/src/actions/actions.types.ts

```diff
@@ -1,2 +1,21 @@
+import { Gear } from '../components/gear/types';
+
+type GearNoId = {
```

Comment:
Wasn't this type added in another PR? And where is the comment on it that was requested?